pyvista/core/filters/data_set.py

```diff
@@ -2269,6 +2730,10 @@ def plot_over_line(dataset, pointa, pointb, resolution=None, scalars=None,
         fname : str, optional
             Save the figure this file name when set.
 
+        Examples
+        --------
+        See the :ref:`plot_over_line_example` example.
```

Comment:
That example is broken too :(

Reply:
This example builds locally for me from a clean build.  I'll be sure to check this pre-release to make sure it got build correctly.

Reply:
Good point. If I run the code blocks manually it works fine. It's likely that something went off the rails during doc building.

I did struggle to get a build: serial runs kept segfaulting without progressing run after run, and one set of parallel builds crashed sphinx in some unexpected way (I got the tracebacks from the worker threads all interleaved in the terminal, amounting to a soup of letters for the most part). This will probably be fine.

Reply:
> I did struggle to get a build: serial runs kept segfaulting without progressing run after run, and one set of parallel builds crashed sphinx in some unexpected way (I got the tracebacks from the worker threads all interleaved in the terminal, amounting to a soup of letters for the most part). This will probably be fine.

Turns out there's still an issue with VTK (dev wheels) with not closing down xwindow connections.  This is going to have to take a bisect to figure out.  Parallel works because it's spreading the build out over several processes, each of which stays under the limit of 256.

As much as I like CI/CD of VTK, it's really meaning that I'm just hunting down different bugs for each of their "patch" releases...

Reply:
Yeah, I figured that this was the issue. But somehow earlier builds would be able to cache the work done up to a segfault, so even with a serial build running `make html` twice would work. I guess with the new autosummary explosion there are just too many plotters being run and it can't reach a point where it can cache before a segfault.

Reply:
> with the new autosummary explosion there are just too many plotters being run and it can't reach a point where it can cache before a segfault.

Exactly.  There are so many figures being built during the documentation build (and not closed out) that it's leading to a segfault.  We're good for now, but if we get more than 2x, even phtml on the CI won't save us and we'll have to find a different workaround.